request/requestid.go

```diff
@@ -0,0 +1,26 @@
+package request
+
+import (
+	"crypto/rand"
+	"fmt"
+
+	"github.com/oklog/ulid"
+)
+
+// ID is an identifier to uniquely identify a request
+type ID struct {
+	timestamp uint64
+	randomID  string
+}
+
```

Comment:
In the future we will need to add a `.Time()` function so we can get the time part from the request-id

Reply:
Issue created